tests/integration/cmd_describe_component_test.go

```diff
@@ -110,54 +131,75 @@ var _ = Describe("odo describe component command tests", func() {
 		for _, label := range []string{
 			helper.LabelNoCluster, helper.LabelUnauth,
 		} {
-			label := label
-
-			It("should describe the component in the current directory", Label(label), func() {
-				By("running with json output", func() {
-					res := helper.Cmd("odo", "describe", "component", "-o", "json").ShouldPass()
-					stdout, stderr := res.Out(), res.Err()
-					Expect(helper.IsJSON(stdout)).To(BeTrue())
-					Expect(stderr).To(BeEmpty())
-					checkDevfileJSONDescription(stdout, "devfile.yaml")
-					helper.JsonPathContentIs(stdout, "runningIn", "")
-					helper.JsonPathContentIs(stdout, "devForwardedPorts", "")
-				})
+			for _, experimental := range []bool{false, true} {
+				label := label
```

Comment:
It would be safer to move label := label before the line 134, so this value could be reused before/after this loop starting in 134.

Reply:
Done in [`2279410` (#6431)](https://github.com/redhat-developer/odo/pull/6431/commits/22794101590ab3824090ab11f06f710f6b967e33)